src/controllers/request-controller.ts

```diff
@@ -9,20 +9,20 @@ import cors from 'cors';
 import { ClassMiddleware, Controller, Get, Post } from '@overnightjs/core';
 
 import CID from 'cids';
-import RequestService from '../services/request-service';
 import { RequestStatus } from '../models/request-status';
 import AnchorService from '../services/anchor-service';
 import { Anchor } from '../models/anchor';
 import { Request } from "../models/request";
 import { inject, singleton } from "tsyringe";
+import RequestRepository from '../repositories/request-repository';
 
 @singleton()
 @Controller('api/v0/requests')
 @ClassMiddleware([cors()])
 export default class RequestController {
 
   constructor(@inject('anchorService') private anchorService?: AnchorService,
-              @inject('requestService') private requestService?: RequestService, ) {
+              @inject('requestRepository') private requestRepository?: RequestRepository, ) {
```

Comment:
I intentionally use `service`s from `controller`s layer. Services can add more logic on top of CRUD repositories. In this case that's not true, but that can change in the future (service layer caching,  etc.).

Reply:
Yep, makes total sense, I just figured that's easy enough to add back if/when we need it.  Having the extra indirection layer now with no logic in it just seemed confusing to me.